pkg/virtctl/vm/migrate.go

```diff
@@ -42,6 +45,8 @@ func NewMigrateCommand() *cobra.Command {
 		Args:    cobra.ExactArgs(1),
 		RunE:    c.migrateRun,
 	}
+
+	cmd.Flags().StringSliceVar(&nodeSlectorLabels, "addedNodeSelector", nil, "--addedNodeSelector=key=value1,key2=value2: configure an additional node selector for the one-off migration attempt. AddedNodeSelector can only restrict constraints already set on the VM. If omitted (recommended!) the scheduler becomes responsible for finding the best Node to migrate the VM to.")
```

Comment:
`By default the scheduler is responsible for finding the best Node, which is the recommended way of migration VMs.`
```suggestion
	cmd.Flags().StringSliceVar(&nodeSlectorLabels, "addedNodeSelector", nil, "--addedNodeSelector=key=value1,key2=value2: configure an additional node selector for the one-off migration attempt. AddedNodeSelector can only restrict constraints already set on the VM. `By default the scheduler is responsible for finding the best Node, which is the recommended way of migrating VMs.)`
```

?

Reply:
nice